mock-core-configs/etc/mock/templates/oraclelinux-7.tpl

```diff
@@ -0,0 +1,56 @@
+# This list is taken from 'epel-7-x86_64' @buildsys-build group, minus the
+# 'epel-*' specific stuff.
+config_opts['chroot_setup_cmd'] = 'install bash bzip2 coreutils cpio diffutils findutils gawk gcc gcc-c++ grep gzip info make patch redhat-rpm-config rpm-build sed shadow-utils tar unzip util-linux which xz'
+
+config_opts['dist'] = 'el7'  # only useful for --resultdir variable subst
+config_opts['releasever'] = '7'
+config_opts['bootstrap_image'] = 'docker.io/library/oraclelinux:7'
+config_opts['package_manager'] = 'yum'
+
+config_opts['yum_install_command'] += "{% if target_arch in ['x86_64', 'ppc64le', 'aarch64'] %} --disablerepo=ol7_software_collections{% endif %}"
```

Comment:
I'm not sure what the purpose of this is? We have an `ol7_software_collections` repo for both `x86_64` and `aarch64` and we don't have any `ppc64le` repos.

Reply:
This is for preventing the SCL repo to influence the bootstrap phase (when not using a bootstrap image) when setting up the root for setting up the build environment.

Reply:
You can probably just drop `ppc64le` completely and leave `x86_64` and `aarch64` based on the intended purpose.

Reply:
Fixed.